src/components/v2/devtronStackManager/DevtronStackManager.component.tsx

```diff
@@ -81,7 +83,7 @@ const ModuleDeailsCard = ({
             <img className="module-details__card-icon mb-16" src={moduleDetails.icon} alt={moduleDetails.title} />
             <div className="module-details__card-name fs-16 fw-6 cn-9 mb-4">{moduleDetails.title}</div>
             <div className="module-details__card-info fs-13 fw-4 cn-7">
-                {moduleDetails.name === 'moreModules' ? (
+                {moduleDetails.name === 'moreIntegrations' ? (
```

Comment:
Can we use 'moreIntegrations' from the util Object?

Reply:
Done